<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RSS</class>
 <widget class="QWidget" name="RSS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RSS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/ico/ico_rss.png</normaloff>:/ico/ico_rss.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QGroupBox" name="grpBox">
     <property name="title">
      <string>RSS Feeds</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QTableView" name="tableView">
        <property name="font">
         <font>
          <underline>false</underline>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>0</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>16</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>16</number>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>TV</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Movies</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Computer</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Console</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>Music</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>XXX</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="rssLayout"/>
        </item>
        <item>
         <widget class="gui::spinner" name="busyspin" native="true">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionRefresh">
   <property name="icon">
    <iconset>
     <normaloff>:/ico/ico_reload.png</normaloff>:/ico/ico_reload.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh RSS feed</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="icon">
    <iconset>
     <normaloff>:/ico/ico_download.png</normaloff>:/ico/ico_download.png</iconset>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="toolTip">
    <string>Download selected items</string>
   </property>
  </action>
  <action name="actionSelectDirectory">
   <property name="icon">
    <iconset>
     <normaloff>:/ico/ico_open.png</normaloff>:/ico/ico_open.png</iconset>
   </property>
   <property name="text">
    <string>Select download directory</string>
   </property>
  </action>
  <action name="actionSaveNZB">
   <property name="icon">
    <iconset>
     <normaloff>:/ico/ico_save.png</normaloff>:/ico/ico_save.png</iconset>
   </property>
   <property name="text">
    <string>Save NZB...</string>
   </property>
   <property name="toolTip">
    <string>Save selected NZB items to disk</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDownloadTo">
   <property name="icon">
    <iconset>
     <normaloff>:/ico/ico_download.png</normaloff>:/ico/ico_download.png</iconset>
   </property>
   <property name="text">
    <string>Download to...</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset>
     <normaloff>:/ico/ico_settings.png</normaloff>:/ico/ico_settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Open nzbs.org settings</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset>
     <normaloff>:/ico/ico_stop.png</normaloff>:/ico/ico_stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>gui::spinner</class>
   <extends>QWidget</extends>
   <header>spinner.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../svnroot/newsflash/gui/extensions/extension_nzbs/tvnzb.qrc"/>
 </resources>
 <connections/>
</ui>
